src/cards/venusNext/SpinInducingAsteroid.ts

```diff
@@ -29,18 +27,6 @@ export class SpinInducingAsteroid extends Card implements IProjectCard {
     });
   }
 
-  public canPlay(player: Player): boolean {
-    if (!super.canPlay(player)) {
-      return false;
-    }
-
-    if (PartyHooks.shouldApplyPolicy(player, PartyName.REDS)) {
-      return player.canAfford(player.getCardCost(this) + REDS_RULING_POLICY_COST * 2, {titanium: true});
```

Comment:
I love it. This code doesn't match all the others by checking the steps raised, which means under rare circumstances this won't be playable. A silent, hard to follow, undetected killer. :D